src/Promitor.Tests.Unit/Builders/Metrics/v1/MetricsDeclarationBuilder.cs

```diff
@@ -1014,9 +1014,12 @@ private AzureMetricConfigurationV1 CreateAzureMetricConfiguration(string azureMe
 
             if (string.IsNullOrWhiteSpace(metricDimension) == false)
             {
-                metricConfig.Dimension = new MetricDimensionV1
+                metricConfig.Dimensions = new List<MetricDimensionV1>
                 {
-                    Name = metricDimension
+                    new()
+                    {
+                        Name = metricDimension
+                    }
```

Comment:
We currently only added basically backwards compatibility to this class.
If any of the tests using this should be extended with multiple dimensions as well, this would need to be further updated.

Reply:
I will add more tests in the coming days